sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1048,7 +1048,7 @@ def _add_argparse_args(cls, parser):
 
 class FlinkRunnerOptions(PipelineOptions):
 
-  PUBLISHED_FLINK_VERSIONS = ['1.7', '1.8', '1.9']
+  PUBLISHED_FLINK_VERSIONS = ['1.7', '1.8', '1.9', '1.10']
```

Comment:
Do we have a process to keep this list up to date?

@mxm @ibzib

Reply:
Unfortunately, we don't. I've suggested to infer this information dynamically from the build but that also caused problems in the past; that's why we reverted to the manual handling. We should document all places which need adjustment.

Reply:
Unfortunately, such a document would itself need to stay up-to-date.

It'd be great if we could have a single source of truth, but it might be somewhat challenging considering the number of different places we need it (Python, Bash, Gradle, etc.)

Reply:
It shouldn't be too hard to have a central file with a list and read that list from these languages. We would have to have some form of build macro here though, to generate a Python file with the versions.

Reply:
Sounds good. I filed BEAM-9541 for this.